PicIt/CameraView.swift

```diff
@@ -0,0 +1,189 @@
+//
```

Comment:
This basically just moves the CameraView into its own file, and out of ContentView.